Bottom: 5bc39e9df0d9e1cd56b012f47e402f0339eee620
Top:    7bca023296c5978649e40b8812dbe639a209a87a
Author: Alexandre Oliva <aoliva@redhat.com>
Date:   2016-05-21 05:05:45 -0300

Refresh of lambda-closures.patch

---

diff --git a/libcc1/libcp1plugin.cc b/libcc1/libcp1plugin.cc
index 3413af2..da5773c 100644
--- a/libcc1/libcp1plugin.cc
+++ b/libcc1/libcp1plugin.cc
@@ -1277,7 +1277,7 @@ plugin_start_new_class_type (cc1_plugin::connection *self,
   tree type = build_named_class_type (RECORD_TYPE, name, base_classes,
 				      ctx->get_source_location (filename,
 								line_number));
-  if ((flags & GCC_CP_CLASS_IS_LAMBDA_CLOSURE))
+  if ((flags & GCC_CP_FLAG_CLASS_IS_LAMBDA_CLOSURE))
     {
       gcc_assert (!name);
       gcc_assert (!base_classes);
